add remote sdk configuration doc (#497)
radarlabs/documentation: docs/sdk/remote-configuration.mdx

---
sidebar_label: Remote Configuration
title: Remote SDK Configuration
description: Documentation for Remote SDK Configuration
---

import Alert from "../../src/components/Alert";

## Remote SDK Configuration

SDK versions 3.14.0 and above have the remote SDK configuration feature enabled, SDK configurations can be accessed through the Radar dashboard.

Remote SDK configurations will override client side SDK configuration.

### Log level

> Override client side log level.

Reviewer: ```suggestion
Set the SDK log level. Overrides any log level set client side.
```

Reviewer: Let's make this after the other sections, to match the order in the dashboard.

### Track once on initialize

> If enabled, the SDK will automatically perform a `Radar.trackOnce` call when the app is launched.

Reviewer: ```suggestion
If enabled, the SDK will automatically perform a `Radar.trackOnce` call when the app is launched or foregrounded.
```

### Start tracking on initialize

> If enabled, the SDK will automatically start tracking with `Radar.startTracking` when the app is launched and tracking is stopped.

Reviewer: ```suggestion
If enabled, the SDK will automatically start tracking with `Radar.startTracking` when the app is launched.
```

## Remote tracking options

SDK tracking options can be set [in dashboard settings](https://radar.com/dashboard/settings?#sdk-settings) for iOS and Android with the three default presets or a custom configuration.

<img src="https://images.ctfassets.net/f2vbu16fzuly/3fTIXlesjMjmtDLlS2yIIc/be9458fa994ae783242e8e5d01cfe697/tracking_options_ios.png" alt="Remote tracking options" />

> [Tracking options](/sdk/tracking) can be targeted by OS (Android and iOS), environment (live and test) and device trip status (on or off trip). By setting tracking options, any client configuration for tracking options set in the SDK will be overridden.

Reviewer: ```suggestion
[Tracking options](/sdk/tracking) can be targeted by OS (Android and iOS), environment (live and test), trip status, and whether the user is in specific geofences. By remotely setting tracking options, any client-side specified tracking options will be overridden.
```
